docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md

```diff
@@ -0,0 +1,284 @@
+- 2D meshes of the inner and outer surfaces for the detection and computation of collisions
+
+The script also generates automatically the geometries we need to attach the finger on the servomotor.
+Also, the script incorporate the maximum dimension specifications described above. **Error messages** will
+pop **in the terminal** in case the desired finger is too big or some geometries cannot be generated.
+To generate a new design of finger, you must:
+
+- Choose a set of design parameters
+- Write their value at the beginning of the code using your favorite code editor
+- Execute the Python script using the command: `python3 mesh_finger.py`
+- Check visually the consistency of the generated part using the now opened Gmsh window
+
+The script will then automatically store the generated mesh files in the `Data` of the starting repository.
+Note that the starting design we propose you to work with is not necessarily the best design nor the
+most classical. There is today a vast library of finger and gripper designs in the soft robot community, and
+finding the optimal one for a given grasping task is not trivial. The design presented here is simple
+enough to give you some intuition about how the parameters influence the finger performances, but
+other designs might work very well also. Don’t hesitate to discuss with the session supervisors about it!
+
+## STEP 3: Simulation of the finger behaviour
+The SOFA scene used to evaluate the elastic behavior of the finger and the contact force with the object
+to grasp is provided in the file `gripper_simulation.py`. In this scene, we make use of the gripper symmetry
+to reduce the overall computation time. As the three servomotors and fingers are at the same distance
+to the object and equally distributed around it, they will apply the same force on it. Because of this, the
+object is likely to stay approximately at the same location before and after grasping. Therefore, we
+idealize the grasping scene by considering that the object is fixed in space and by simulating the behavior
+of one finger only.
+
+<img src="images\simulation.png" width="400" title="Figure 4: Screenshot of the SOFA scene for the soft finger simulation" alt="Figure 4: Screenshot of the SOFA scene for the soft finger simulation"/>
+
+The scene contains:
+- The 3D mesh of the finger along with an elastic force field to simulate its deformation
+- A box with rigidified mesh nodes at the left wall base, which pose is updated according to the
+servomotor rotation
+- A box of rigidified mesh nodes at the right wall base which pose is fixed
+- A mesh of the object to grasp, fixed in space.
+- Collision models to account for eventual collisions between the finger and the object, and
+between the inner surfaces of the left and right walls
+
+<pre>
+<a href="details/procedural-finger.py"> <img src="../../../images/icons/play.png" width="14px"/>Try the scene in SOFA.</a>
```

Comment:
If I understand correctly here it should be gripper_simulation.py?

Reply:
Ow I didn't see that, yes it should!